knowledge-base/actions/endbug/add-and-commit/action-security.yml

```diff
@@ -0,0 +1,2 @@
+name: Add & Commit
+#Note: github token not used
```

Comment:
@vandana41 this uses `github-token` [check_here](https://github.com/EndBug/add-and-commit/blob/main/action.yml#L64), please do the analysis